phoenix-core/src/main/java/org/apache/phoenix/schema/stats/DefaultStatisticsCollector.java

```diff
@@ -121,92 +110,69 @@
             cachedGuidePosts = null;
         }
     }
-    
-    private void initGuidepostDepth() throws IOException, ClassNotFoundException, SQLException {
-        // First check is if guidepost info set on statement itself
+
+    @Override
+    public void init() throws IOException {
+        try {
+            initGuidepostDepth();
+            initStatsWriter();
+        } catch (SQLException e) {
+            throw new IOException(e);
+        }
+        LOG.info("Initialization complete for " +
+                this.getClass() + " statistics collector for table " + tableName);
+    }
+
+    /**
+     * Determine the GPW for statistics collection for the table.
+     * The order of priority is as follows
```

Comment:
Could you highlight the the order of priority is listed from high to low as follows?